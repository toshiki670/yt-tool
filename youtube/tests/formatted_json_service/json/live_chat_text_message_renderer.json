{
  "clickTrackingParams": "CAEQl98BIhMIu6L3wb27igMVi9Y0Bx2m8wVM",
  "replayChatItemAction": {
    "actions": [
      {
        "clickTrackingParams": "CAEQl98BIhMIu6L3wb27igMVi9Y0Bx2m8wVM",
        "addChatItemAction": {
          "item": {
            "liveChatTextMessageRenderer": {
              "message": { "runs": [{ "text": "ふっふー！" }] },
              "authorPhoto": {
                "thumbnails": [
                  {
                    "url": "https://yt4.ggpht.com/Ts81zNr4SlA4qVIkemILzivohyBLqMaNjUwnCyH-p4o7CheMhkdPhIaGf89o1N1KWwVfkUjEczs=s32-c-k-c0x00ffffff-no-rj",
                    "width": 32,
                    "height": 32
                  },
                  {
                    "url": "https://yt4.ggpht.com/Ts81zNr4SlA4qVIkemILzivohyBLqMaNjUwnCyH-p4o7CheMhkdPhIaGf89o1N1KWwVfkUjEczs=s64-c-k-c0x00ffffff-no-rj",
                    "width": 64,
                    "height": 64
                  }
                ]
              },
              "contextMenuEndpoint": {
                "clickTrackingParams": "CAEQl98BIhMIu6L3wb27igMVi9Y0Bx2m8wVM",
                "commandMetadata": {
                  "webCommandMetadata": { "ignoreNavigation": true }
                },
                "liveChatItemContextMenuEndpoint": {
                  "params": "Q2g0S0hBb2FRMHR4Y2pndE5uRjFORzlFUm1OVVduZG5VV1JrTVVWTmVYY2FLU29uQ2hoVlExcHNSRmg2UjI5dk4yUTBOR0ozWkU1UFlrWmhZMmNTQzBsb1NsY3RXVkJLTFhvd0lBRW9BVElhQ2hoVlEycDRRV0ZKY3pKdmIxSk5hVFZrY0VOSFMwVndZWGM0QWtnQlVBRSUzRA=="
                }
              },
              "id": "ChwKGkNLcXI4LTZxdTRvREZjVFp3Z1FkZDFFTXl3",
              "timestampUsec": "1734869056809328",
              "authorExternalChannelId": "UCjxAaIs2ooRMi5dpCGKEpaw",
              "contextMenuAccessibility": {
                "accessibilityData": { "label": "Chat actions" }
              },
              "timestampText": { "simpleText": "4:16" },
              "trackingParams": "CAEQl98BIhMIu6L3wb27igMVi9Y0Bx2m8wVM"
            }
          },
          "clientId": "CKqr8-6qu4oDFcTZwgQdd1EMyw"
        }
      }
    ],
    "videoOffsetTimeMsec": "256333"
  }
}
