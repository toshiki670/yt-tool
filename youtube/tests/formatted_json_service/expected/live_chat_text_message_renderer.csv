id,authorExternalChannelId,postedAt,category,authorName,content
ChwKGkNLcXI4LTZxdTRvREZjVFp3Z1FkZDFFTXl3,UCjxAaIs2ooRMi5dpCGKEpaw,2024-12-22T21:04:16.809328+09:00,Text message,,message: 'ふっふー！'
